### DISCUSSION TITLE
PhpStan Error on Collection

### DISCUSSION THREAD

**Discussion by adalessa** [spatie/laravel-data#26] [Q&A] [answered]
I have started using laravel data, but running phpstan I have an error about the type not being compatible
```
  14     PHPDoc type for property App\Services\Product::$images with type
         array<App\Services\Image> is incompatible with native type
         Spatie\LaravelData\DataCollection.
```
Of course this can be skipped on the configuration, but want to be sure if it is something know since I haven't see any discussion about this.
Thanks

**Comment by rubenvanassche** [comment, marked as answer]
Hi @adalessa,

Could you try `Spatie\LaravelData\DataCollection<App\Services\Image>` as an annotation? Version 1.1.0 we released yesterday has support for this and PHPStan should understand it.

**Reply by adalessa** [reply]
Thank you very much for the response. That's solve it
